src/app/layout/header/dialog-user/dialog-user.component.ts

```diff
@@ -59,21 +50,14 @@ export class DialogUserComponent implements OnInit, OnDestroy {
       return;
     }
     this.store.dispatch(
-      userActions.updateUserData({ userData: this.getProfile() })
+      UserClaimActions.updateUserData({ userData: this.getProfile() })
     );
   }
 
   private getProfile() {
     const profileData = this.profileForm.getRawValue();
     return {
       ...profileData,
-      birthdate: profileData.birthdate.getTime(),
     };
   }
-
-  private getLegalAge() {
```

Comment:
Well, you do not need to be adult to use metamask and buy tokens. Metamask do not check identity (you only need credit card for example from parents :D ). Anyway it was only checked when adding birthday. However birthday and address is no longer needed in user data form, so i removed all connected code. If this will be needed we can always revert it back by checking git history :)